helm_deploy/values-prod.yaml

```diff
@@ -0,0 +1,19 @@
+---
+# Per environment values which override defaults in approved-premises-ui/values.yaml
```

Comment:
Copy and paste error in comment? `approved-premises-ui` should be temporary-accommodation-ui`

Reply:
Yeah good spot, that should be in all 4 helm charts so I'll add another commit to rename them 👍